doc(release): Prepare release notes for modules next 22.10 (#2198)
centreon/centreon-documentation: versioned_docs/version-22.10/releases/centreon-commercial-extensions.md

## Centreon MAP

### 22.10.5

Release date: `soon`

Release date: `soon`

> You need to [execute the **configure.sh** script](../graph-views/map-web-install.md#étape-6--exécuter-le-script-configuresh) in order to apply some fixes provided by this release.

#### Bug fixes
- [API] Fixed an issue with create element in view that could break default icon.
- [CEIP] Added more details for MAP engine telemetry.
- [Editor] Fixed an issue in shapes edition modal that caused scrollbar to be reset when browsing images.
- [Editor] Fixed view edition with some links having corrupted bend points.

> - [Editor] Update draw.io library.

Reviewer: To rephrase

- [Install] Fixed an issue preventing the installation of centreon-map-engine on the same machine as centreon-map-server.
- [Monitoring] Fixed an issue where MAP was not supporting performance data returned  by Cisco ASA plugin.

> - [Server] Fix migration issue with images that could stop migration process.

Reviewer: ```suggestion
- [Server] Fixed a migration issue with images that could stop the migration process.
```

- [Server] Fixed an issue that caused server to crash on corrupted metrics.
- [Server] Fixed an issue that prevent the migration between MAP legacy and MAP.
- [Server] Fixed an issue that prevented access to MAP in HTTPS.
- [Server] Fixed an issue to allow more than 25 maps to be migrated.
- [Server] Fixed an issue with authentication when several sessions are active.
- [Server] Fixed an issue with insert an object in a link prevented the map from being saved.
- [Server] Fixed an issue with map sharing.
- [Server] Fixed configure.sh script to include storage event from broker output.

> - [Server] Fixed issue that made new hosts and services unavailable until restart.

Reviewer: ```suggestion
- [Server] Fixed an issue that made new hosts and services unavailable until restart.
```

> - [Server] Fixed the server startup error due to PF without virtual hosts.

Reviewer: ```suggestion
- [Server] Fixed an issue with the server startup error caused by platform without virtual hosts.
```

> - [UI] Fixed Colors for a "Text" type element is ignored by the viewer.

Reviewer: ```suggestion
- [Viewer] Fixed an issue with colors for a "Text" type element that was ignored by the viewer.
```

> - [UI] Fixed an issue when tooltip don't display for "old perfdata" plugin in the graph.

Reviewer: ```suggestion
- [UI] Fixed an issue where tooltips were not displayed for "old perfdata" plugin in the graph.
```

> - [UI] Fixed an issue when updating a resource that did not display the new resource name.

Reviewer: ```suggestion
- [UI] Fixed an issue where updating a resource was not displaying the new resource name.
```

- [UI] Fixed an issue with Host and Service icons configuration that were not handled properly.
- [UI] Fixed bug when custom URI is set, some old media in map are not displayed correctly.

> - [UI] Fixed bug who display a blank page when user click on host name inside a map.

Reviewer: ```suggestion
- [UI] Fixed an issue where a blank page was displayed when the user clicked on a host name inside a map.
```

> - [UI] Fixed issue when Dark Theme is not apply on tooltip in Geo Views / MAP ENGINE.

Reviewer: ```suggestion
- [UI] Fixed an issue where dark theme was not applied on tooltips in Geo views.
```

- [UI] Fixed issues and improved the map display (visibility, background and breadcrumb color).

> - [UI] Fixed the background layout not fitting large map.

Reviewer: ```suggestion
- [UI] Fixed the background layout that was not fitting large maps.
```

- [Viewer] Fixed an issue that caused the unit not matching the axis on graph widget.

#### Enhancements

> - [API] Improve api response when server is starting up.

Reviewer: ```suggestion
- [API] Improved the API response when server is starting up.
```

> - [Editor] Manage font properties in editor.

Reviewer: ```suggestion
- [Editor] You can now manage font properties in the editor.
```

## Centreon MAP

### 22.10.5

> Release date: `April 18, 2023`

Author: ```suggestion
Release date: `April 24, 2023`
```

> > You need to [execute the **configure.sh** script](../graph-views/map-web-install.md#étape-6--exécuter-le-script-configuresh) in order to apply some fixes provided by this release.

Reviewer 2: ```suggestion
> You need to [execute the **configure.sh** script](../graph-views/map-web-install.md#step-6--execute-the-configuresh-script) in order to apply some fixes provided by this release.
```

## Centreon MAP Legacy

### 22.10.5

> Release date: `April 18, 2023`

Author: ```suggestion
Release date: `April 24, 2023`
```

## Centreon BAM

### 22.10.2

> Release date: `April 18, 2023`

Author: ```suggestion
Release date: `April 24, 2023`
```

## Centreon Auto Discovery

### 22.10.1

Release date: `soon`

> - [UI] Fixed a cosmetic issue in Bam & Autodiscovery table header

Reviewer: ```suggestion
- [UI] Fixed a cosmetic issue in BAM and Autodiscovery table header.
```

## Centreon Auto Discovery

### 22.10.1

> Release date: `April 18, 2023`

Author: ```suggestion
Release date: `April 24, 2023`
```

## Centreon Anomaly Detection

### 22.10.2

> Release date: `April 18, 2023`

Author: ```suggestion
Release date: `April 24, 2023`
```
